libc++ doesn't like "using namespace std" in C++11

network.cc:466:59: error: invalid operands to binary expression ('__bind<int &, sockaddr *, unsigned long>' and 'int')
  if (bind(netsocket,(struct sockaddr *) &sin,sizeof(sin))==-1) {
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^ ~~

--- network.cc.orig	2008-02-22 15:51:22 UTC
+++ network.cc
@@ -463,7 +463,7 @@ int IncomingConnection::createSocket() {
   sin.sin_addr.s_addr = INADDR_ANY;
   sin.sin_port = htons(Port);
     
-  if (bind(netsocket,(struct sockaddr *) &sin,sizeof(sin))==-1) {
+  if (::bind(netsocket,(struct sockaddr *) &sin,sizeof(sin))==-1) {
     snprintf(errorMessage,128,_("Unable to bind on port %d."),Port);
     return(-1);
   }
